create-keystone-app/starter/README.md

```diff
@@ -0,0 +1,50 @@
+# Keystone Project Parter
+
+Welcome to Keystone!
+
+Run
+
+```
+npm keystone dev
```

Comment:
I think we could add `yarn dev` here as another option as we lead with it in the project README.md - https://github.com/keystonejs/create-keystone-app/blob/noviny/improvements/README.md

Perhaps:

---

To get started, run:

```yarn dev```

or

```npm run dev```